src/server/test/web/readingsBarGroupQuantity.js

```diff
@@ -122,87 +127,113 @@ mocha.describe('readings API', () => {
                     const expected = await parseExpectedCsv('src/server/test/web/readingsData/expected_bar_group_ri_15-20_mu_kWh_gu_kWh_st_-inf_et_inf_bd_76.csv');
                     // Create a request to the API for unbounded reading times and save the response
                     const res = await chai.request(app).get(`/api/unitReadings/bar/groups/${GROUP_ID}`)
-                        .query({ 
-                            timeInterval: ETERNITY.toString(), 
+                        .query({
+                            timeInterval: ETERNITY.toString(),
                             barWidthDays: '76',
-                            graphicUnitId: unitId });
+                            graphicUnitId: unitId
+                        });
                     // Check that the API reading is equal to what it is expected to equal
                     expectReadingToEqualExpected(res, expected, GROUP_ID);
                 });
-                // Add BG7 here
+                
```

Comment:
OED generally leaves at most one blank line between tests. Having two and containing spaces at the front is less desirable. Could you please remove them. There is also two empty blank lines after the test to remove.